constants/issues.ts

```diff
@@ -0,0 +1,44 @@
+export type IssueStatus = 'todo' | 'doing' | 'done'
+export interface Issue {
+  id: string
+  title: string
+  labels: IssueStatus[]
```

Comment:
A question on this, do we want to use labels exclusively for statuses?

How I envisioned using labels was in a format like the following `[prefix]-[key]-[value]` (e.g. `pb-status-doing`)

- prefix: used to indicate that this label is managed by the planning boards (therefore `pb` `p(lanning)b(oards)` or `rpb` `r(adicle)p(lanning)b(oards)`)

- key: what the label represents (in this case `status`, but it also leaves us room to implement other metadata into the issues e.g. ordering -> `pb-index-0`)

- value: self explanatory

This way our labels can co-exist, with user generated ones.

This might be over-engineering on my part, and is definitely out of scope of the PR, but I just want to hear your thoughts.

cc @maninak @carboniq-dev

Reply:
Yes, that's pretty much how I envisioned it too.

Regarding the prefix/namespace I'd stick to `RPB`.

The separator would need to be something less conflicting than `-`.

All of this I see as out of scope for this UI issue, but otherwise super important for the prototype phase, yes.